dotnet/src/Connectors/Connectors.AssemblyAI/AssemblyAIServiceCollectionExtensions.cs

```diff
@@ -0,0 +1,85 @@
+﻿// Copyright (c) Microsoft. All rights reserved.
+
+using System;
+using System.Net.Http;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.SemanticKernel.AudioToText;
+using Microsoft.SemanticKernel.Connectors.AssemblyAI;
+using Microsoft.SemanticKernel.Http;
+
+namespace Microsoft.SemanticKernel;
+
+/// <summary>
+/// Provides extension methods for <see cref="IServiceCollection"/> and related classes to configure AssemblyAI connectors.
+/// </summary>
+public static class AssemblyAIServiceCollectionExtensions
+{
+    /// <summary>
+    /// Adds the AssemblyAI audio-to-text service to the kernel.
+    /// </summary>
+    /// <param name="builder">The <see cref="IKernelBuilder"/> instance to augment.</param>
+    /// <param name="apiKey">AssemblyAI API key, <a href="https://www.assemblyai.com/dashboard">get your API key from the dashboard.</a></param>
+    /// <param name="endpoint">The endpoint URL to the AssemblyAI API.</param>
+    /// <param name="serviceId">A local identifier for the given AI service.</param>
+    /// <param name="httpClient">The HttpClient to use with this service.</param>
+    /// <returns>The same instance as <paramref name="builder"/>.</returns>
+    public static IKernelBuilder AddAssemblyAIAudioToText(
+        this IKernelBuilder builder,
+        string apiKey,
+        string? endpoint = null,
+        string? serviceId = null,
+        HttpClient? httpClient = null
+    )
+    {
+        Verify.NotNull(builder);
+        AddAssemblyAIAudioToText(builder.Services, apiKey, endpoint, serviceId, httpClient);
+        return builder;
+    }
+
+    /// <summary>
+    /// Adds the AssemblyAI audio-to-text service to the list.
+    /// </summary>
+    /// <param name="services">The <see cref="IServiceCollection"/> instance to augment.</param>
+    /// <param name="apiKey">AssemblyAI API key, <a href="https://www.assemblyai.com/dashboard">get your API key from the dashboard.</a></param>
+    /// <param name="endpoint">The endpoint URL to the AssemblyAI API.</param>
+    /// <param name="serviceId">A local identifier for the given AI service.</param>
+    /// <param name="httpClient">The HttpClient to use with this service.</param>
+    /// <returns>The same instance as <paramref name="services"/>.</returns>
+    public static IServiceCollection AddAssemblyAIAudioToText(
+        this IServiceCollection services,
+        string apiKey,
+        string? endpoint = null,
+        string? serviceId = null,
+        HttpClient? httpClient = null
```

Comment:
@Krzysztof318 It's only for `IServiceCollection` extension methods, since we expect HttpClient to be configured in service collection. While for `IKernelBuilder` extension methods we should accept `HttpClient` as a parameter.

Reply:
I see, in that case, I added the parameter back to the kernel builder extension, but not the service collection extension.